pkg/instance/handler.go

```diff
@@ -63,8 +66,17 @@ type CreateInstanceRequest struct {
 //   403: Error
 //   404: Error
 //   415: Error
-func (h Handler) Create(c *gin.Context) {
-	var request CreateInstanceRequest
+func (h Handler) Launch(c *gin.Context) {
+	deploy := true
+	if deployParam, ok := c.GetQuery("deploy"); ok {
```

Comment:
I would prefer having `deploy` in the JSON body as well instead of mixing body/query params.

Reply:
That's where I had it initially but I felt it didn't really belong there. It's metadata compared to the other properties but let's have a chat